src/LanguageServer.SemanticModel.Xml/XmlSyntaxHelper.cs

```diff
@@ -93,7 +93,7 @@ public static IEnumerable<SyntaxNode> DescendantNodes(this SyntaxNode syntaxNode
                 yield return childNode;
 
                 foreach (SyntaxNode descendantNode in childNode.DescendantNodes())
-                    yield return childNode;
+                    yield return descendantNode;
```

Comment:
I am 99% sure this is was not correct behaviour before change

Reply:
Yep, nice catch!